packages/http-server/src/server.ts

```diff
@@ -108,6 +108,13 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
             reply.headers(output.headers);
           }
 
+          response.validations.output.forEach(
+            validation =>
+              validation.severity === DiagnosticSeverity.Error
+                ? request.log.error(validation.message)
+                : request.log.warn(validation.message),
```

Comment:
You should add handling for other severities (Informational, Hint). I checked out of curiosity whether we use them currently and we don't, but this will create minor bug if we start.